pkg/reconciler/reconciler.go

```diff
@@ -846,7 +857,50 @@ func (r *Reconciler) doUninstall(actionClient helmclient.ActionInterface, u *upd
 		if log.V(4).Enabled() {
 			fmt.Println(diff.Generate(resp.Release.Manifest, ""))
 		}
+
+		if !wait {
+			updater.EnsureCondition(conditions.Deployed(corev1.ConditionFalse, conditions.ReasonUninstallSuccessful, ""))
+		}
+	}
+
+	if wait {
+		updater.EnsureCondition(conditions.Deployed(corev1.ConditionFalse, conditions.ReasonUninstallSuccessful, "Waiting until all resources are deleted."))
```

Comment:
Should we have a new `Reason`, something like `ReasonUninstallProgressing` or `ReasonUninstallWaiting`?  Not sure exactly what the right terminology is for "we started the uninstall, but it won't finish until all of the underlying resources are actually gone"